### DISCUSSION TITLE
0DE4120B-CA29-717F-FD6A-69BF84C77F6A

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#23] [General]
# 0DE4120B-CA29-717F-FD6A-69BF84C77F6A

Kepler-11 is a Sun-like star in the constellation Cygnus, 2,148 light-years away from Earth. The discovery of 6 planets around this star was announced on February 2, 2011, based on the analysis of data from the Kepler space telescope. It is one of the flattest and most compact planetary systems ever discovered. All of its planets are larger than Earth, the largest being the size of Neptune.

https://celestia.mobi/addon?item=0DE4120B-CA29-717F-FD6A-69BF84C77F6A

**Comment by exocritic5457** [comment]
Wow, those are some REALLY rocky neptunians

**Reply by exocritic5457** [reply]
Rocky they are no longer. A1
